Makefile

```diff
@@ -19,6 +19,7 @@ default: help
 VERSION ?= 1.4.0
 RELEASE_SRC = apache-apisix-ingress-controller-${VERSION}-src
 LOCAL_REGISTRY="localhost:5000"
+CUSTOM_REGISTRY ?= ""
```

Comment:
Instead of using a new environment variable, can we change the old one to a more generic name? The new variable does not seem necessary.

Reply:
+1

Reply:
Thanks for review @tao12345666333 @lingsamuel, I've tried renaming REGISTRY, here I'm worried if the kind-up part is needed, I think we should check if REGISTRY points to kind and then run the judgment?

Reply:
IMO, it would be better to move the kind-up from the prerequisite of push-images to the e2e-test. Since the users may indeed have a registry at localhost:5000.
Users can then run the e2e like this: build the images, export the REGISTRY env var, run `make push-images`, and then execute ginkgo directly.
Or we could provide a new entry such as `e2e-test-local` just like the e2e-test but without kind-up.

Reply:
> IMO, it would be better to move the kind-up from the prerequisite of push-images to the e2e-test. Since the users may indeed have a registry at localhost:5000. Users can then run the e2e like this: build the images, export the REGISTRY env var, run `make push-images`, and then execute ginkgo directly. Or we could provide a new entry such as `e2e-test-local` just like the e2e-test but without kind-up.

@lingsamuel Thanks for the suggestion, e2e-test-local has been re-added.